server/issue.go

```diff
@@ -633,12 +633,12 @@ func httpAPIAttachCommentToIssue(ji Instance, w http.ResponseWriter, r *http.Req
 			mattermostName, jiraName, mime, e := client.AddAttachment(api, attach.IssueKey, fileId, conf.maxAttachmentSize)
 			if e != nil {
 				notifyOnFailedAttachment(ji, mattermostUserId, attach.IssueKey, e, "file: %s", mattermostName)
-			}
-
-			if isImageMIME(mime) || isEmbbedableMIME(mime) {
-				extraText += "\n\nAttachment: !" + jiraName + "!"
 			} else {
-				extraText += "\n\nAttachment: [^" + jiraName + "]"
+				if isImageMIME(mime) || isEmbbedableMIME(mime) {
```

Comment:
I believe you could add a `continue` after the `notifyOnFailedAttachement` call and not have to introduce the new `else` block

Reply:
Right! Done